backend/medtagger/api/scans/business.py

```diff
@@ -147,19 +147,20 @@ def _validate_label_elements(elements: List[Dict], files: Dict[str, bytes]) -> N
                 raise InvalidArgumentsException(message.format(label_element['image_key']))
 
 
-def add_label(scan_id: ScanID, elements: List[Dict], files: Dict[str, bytes],
+def add_label(scan_id: ScanID, task_id: TaskID, elements: List[Dict], files: Dict[str, bytes],
```

Comment:
Maybe there should be a Task's key? Hmmm...

BTW. We should think about IDs vs. Keys (in another Story).

Reply:
Yes, we should use keys everywhere where it is possible